src/commons.jl

```diff
@@ -0,0 +1,196 @@
+#-----------------------------------------------------------
+#-----------------------------------------------------------
+# Define types and base functions to access windows registry
+# -- part of WinReg Module --
+#----------------------------------------------------------
+#----------------------------------------------------------
+
+
+typealias HKEY UInt32 #DWORD also on 64-bit platforms
```

Comment:
I intended put in `wintypes.jl` general Win32 types that are not directly related to registry functions, and in `commons.jl` all function / types used to manipulate registry

But I'll hear your suggestions about alternative ways to reorder code

Leonardo